qa/Rakefile

```diff
@@ -0,0 +1,66 @@
+require 'rspec'
```

Comment:
Can you add these lines to this file? This will make sure the code actually read the `Gemfile.lock` and the the `.bundler/config`, This is needed if you want to run `bundle install --path vendor`

```ruby
require "rubygems" # Might not be required on all ruby, but doesnt hurt.
require "bundler/setup"
```